lib/view_model/unspent_coins/unspent_coins_list_view_model.dart

```diff
@@ -27,13 +27,17 @@ abstract class UnspentCoinsListViewModelBase with Store {
   ObservableList<UnspentCoinsItem> get items => ObservableList.of(bitcoin.getUnspents(wallet).map((elem) {
       final amount = bitcoin.formatterBitcoinAmountToString(amount: elem.value) +
           ' ${wallet.currency.title}';
+  
+     final info = _unspentCoinsInfo.values
+          .firstWhere((element) => element.walletId.contains(wallet.id) &&
+          element.hash.contains(elem.hash));
```

Comment:
Better check is string equal than contains for both cases

Reply:
Okay, Sir

Reply:
@mkyq